website/pages/docs/automatic.mdx

```diff
@@ -1,33 +1,31 @@
 ---
-title: 'Automatic Mode'
+title: 'How to use Automatic Mode'
 description: 'The default mode of Million.js'
 ---
 
 import { Callout, Tabs, Tab, Steps } from 'nextra-theme-docs';
 
 # Automatic Mode
 
-Automatic mode is the default mode of Million.js. It automatically converts React components to Million.js components.
+Million.js comes with Automatic Mode as its default setting. This transforms your React components into Million.js components, improving speed without any major code changes.
 
-### Ignoring components
 
-If a certain component conflicts with the Million.js runtime, it's possible to skip over them via the `// million-ignore` comment.
+When [installing Million into your project](/docs/install), automatic mode is recommended for most users. This passes the `auto` option into the compiler.
```

Comment:
This has been handled, thanks!


When [installing Million into your project](/docs/install), automatic mode is recommended for most users and this is because it packs in it all the necessary optimizations that your application needs. The `auto` option of the compiler toggles it.